Mesh_3/include/CGAL/Mesh_edge_criteria_3.h

```diff
@@ -121,58 +122,38 @@ class Mesh_edge_criteria_3
 
   /// \name Creation
   /// @{
+
   /*!
   * returns an object to serve as criteria for edges.
   *
   * \param length_bound is an upper bound
   * for the length of the edges which are used to discretize the curves.
+  * It can be a functional or a constant.
   * \param min_length_bound is a desired lower bound
   * for the length of the edges which are used to discretize the curves.
   * Only edges that are longer than this bound will be refined. Using
   * this lower bound can be handy on some domains, but using it may
   * break all the surface topology guarantees of the meshing algorithm.
   * It is not guaranteed to be exactly respected in the output mesh.
+  * \param distance_bound is an upper bound for the distance from the
+  * edge to the corresponding 1D feature.
+  * It can be a functional or a constant.
   *
   * \note If one parameter is set to 0, then its corresponding criterion is ignored.
-  */
-  Mesh_edge_criteria_3(const FT& length_bound,
-                       const FT& min_length_bound = 0)
-
-    : p_size_(new Mesh_3::internal::Sizing_field_container<
-                Mesh_constant_domain_field_3<GT,Index> ,
-                FT,
-                Point_3,
-                Index>(length_bound))
-    , min_length_bound_(min_length_bound)
-  {}
-
-  // Nb: SFINAE to avoid wrong matches with built-in numerical types
-  // as int.
-
-  /*!
-  * returns an object to serve as criteria for edges.
-  * The behavior and semantic of the argument are the same
-  * as above, except that the `length_bound`
-  * parameter is a functional instead of a constant.
   *
-  * @tparam SizingField a model of `MeshDomainField_3`
+  * @tparam SizingField and @tparam DistanceField are either scalars
```

Comment:
The formatted documentation looks strange.

https://cgal.github.io/7532/v3/Mesh_3/classCGAL_1_1Mesh__edge__criteria__3.html#abf92d6d117698a5194a23970f69804e6